### ISSUE TITLE
Docker fails

### ISSUE THREAD

**Issue by rcemper** [caretdev/grafana-intersystems-datasource#1]
D:\GitHub\grafana-intersystems-datasource>docker-compose up -d
[+] Running 1/1
 ✘ iris Error                                                                                                1.0s
Error response from daemon: Head "https://registry-1.docker.io/v2/store/intersystems/iris-community/manifests/2020
.4.0.524.0": unknown: store has been removed

**Comment by daimor** [comment]
Yeah, the project is outdated a bit, requires some time to update
